piperider_cli/compare_report.py

```diff
@@ -271,3 +256,8 @@ def output_report(directory):
 
         console.print()
         console.print(f"Comparison report: {filename}")
+
+        if debug:
```

Comment:
Im curious, why do we need a flag to generate `comparison_data.json`?
I ask this because we already generate `run.json`...
Is there some product context that's driving this decision to not output it by default?

Reply:
`comparison_data.json` should be a temporary workaround just let FE local testing can work now. Due to compare report is based on two `run.json`. It's better to always merge 2 `run.json` files on the fly.